discord-script.py

```diff
@@ -1,19 +1,67 @@
+import random
 import discord
-from discord.ext import commands
+from discord import app_commands
 
-# discord 2.0 requires intents
 
-bot = commands.Bot(command_prefix="!", intents=discord.Intents.all())
+class DiscordBot(discord.Client):
+    def __init__(self):
+        super().__init__(intents=discord.Intents.default())
+        self.synced = False
+
+    # bot syncs slash commands on startup
+    async def on_ready(self):
+        await tree.sync(guild=discord.Object(
+            id=1030163730538954853))  # without this the slash commands will not instantly update (encrypt later)

```

Comment:
The id parameter's value should be converted into a variable since it's used several times throughout the file.

Reply:
The server ID variable has been added.